### ISSUE TITLE
load_test.bash complicated usage by passing $1 as Command due to card reader selection

### ISSUE THREAD

**Issue by psytester** [BeID-lab/eIDClientCore#92]
The load_test.bash usage is to complicated, because the $1 is passed as final $CMD

Since the card Reader selection requires the Name of the Reader, I have to use spaces in its Name and finally the whole string is not passed correctly to $CMD

```
./bin/load_test.bash "env LD_LIBRARY_PATH=..... Start_Testcase -t ... --service_provider=https://..... -p 123456 --card-reader=\"Virtual PCD 00 00\" --cancel-after-paos" 5
```

The "--card-reader=Virtual PCD 00 00" with its spaces breaks the call and eIDClientCore does not get the full reader string.

Another issue is, that the load_test.bash uses only one card ready, which may be bussy by another thread

**Comment by frankmorgner** [comment]
When you are doing the escaping correct, this will work. However, I don't get those things right either! You could try wrapping the reader's name into `'`...

**Comment by eriknellessen** [comment]
Solved in 1959490f54a148a6631e1dd21b2065d5d795d4e3.
Example usage:
`LD_LIBRARY_PATH=$PWD/lib:$PWD/lib64:$LD_LIBRARY_PATH ./bin/load_test.bash 20 ./bin/Start_Testcase -t Aute -p 123456 --card-reader="Virtual PCD 00 00" --cancel-after-paos
`
